extensions/resteasy-server-common/deployment/src/main/java/io/quarkus/resteasy/server/common/deployment/ResteasyServerCommonProcessor.java

```diff
@@ -279,6 +284,27 @@ public void build(
                 }
             }
         }
+        // look for all implementations of abstract classes annotated @Path
+        do {
+            Set<DotName> tmpAbstract = new HashSet<>();
+            for (final DotName iface : pathAbstract) {
+                final Collection<ClassInfo> implementors = index.getAllKnownSubclasses(iface);
```

Comment:
Given you get all the known subclasses here, I fail to see while you need the outer while loop?

This method also includes the non-direct subclasses.

Reply:
concrete class of subclass of subclass